typeorm-seeding.config.ts

```diff
@@ -0,0 +1,18 @@
+// TODO: add type
```

Comment:
Remove this `TODO`.

Reply:
Resolved on 298ab1e4e2ae083bd4d5450e9bc709597987671c.